metadata-ingestion/src/datahub/ingestion/source/snowflake/snowflake_config.py

```diff
@@ -308,6 +308,13 @@ class SnowflakeV2Config(
         " assertions CLI in snowflake",
     )
 
+    pushdown_deny_usernames: List[str] = Field(
```

Comment:
we should think about how we can avoid needing to specify every config twice - not sure if there's a good way to do it though

Reply:
While everything is possible, those refactors will be easier when we remove the code base for queries v1.